pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -57,6 +57,11 @@ type Controller struct {
 	informer       cache.SharedIndexInformer
 	addCallback    addSecretCallback
 	removeCallback removeSecretCallback
+	// wachedNamespace
```

Comment:
Remove these or add a TODO comment on why they are listed.

Reply:
Opps - I thought I did remove these will take care of it